e2e/cypress/integration/accessibility/accessibility_dropdowns_spec.js

```diff
@@ -103,8 +103,8 @@ describe('Verify Accessibility Support in Dropdown Menus', () => {
     });
 
     it('MM-T1477 Accessibility Support in Status Dropdown', () => {
-        // # Press tab from Add Team button
-        cy.get('#create_teamTeamButton').focus().wait(TIMEOUTS.HALF_SEC).tab({shift: true}).tab().tab();
+        // # Press tab from the test team button
+        cy.get(`#${testTeam.name}TeamButton`).focus().wait(TIMEOUTS.HALF_SEC).tab({shift: true}).tab().tab().tab();
```

Comment:
This is to ensure it'll pass on Cloud as well.

Reply:
@saturninoabril 
One of test case is failing at my end consistently. I think we are verifying class for next team

![image](https://user-images.githubusercontent.com/6909890/101042087-c726e880-354a-11eb-9d06-de0096fd2ecf.png)

Reply:
Can you run with `export CYPRESS_resetBeforeTest=true`? It's set to false by default to not break/destroy local setup (in case a dev is working on other things) but in CI, we have it set to true.